src/EnergyPlus/AirflowNetwork/src/Elements.cpp

```diff
@@ -1512,10 +1505,10 @@ namespace AirflowNetwork {
 
         // Get component properties
         DifLim = 1.0e-4;
-        Width = state.dataAirflowNetwork->MultizoneSurfaceData(IL).Width;
-        Height = state.dataAirflowNetwork->MultizoneSurfaceData(IL).Height;
-        Fact = state.dataAirflowNetwork->MultizoneSurfaceData(IL).OpenFactor;
-        Loc = (state.dataAirflowNetwork->AirflowNetworkLinkageData(IL).DetOpenNum - 1) * (NrInt + 2);
+        Width = state.afn->MultizoneSurfaceData(IL).Width;
```

Comment:
This is just a lot more readable.  :+1:

Reply:
Yes, I've left the namespace as is because most access of these will be through the object now. There is still some repetition of `AirflowNetwork` around, but this PR is already so big I'd rather leave that for another day.